src/arithmetic/bound_deducer.cc

```diff
@@ -154,8 +156,12 @@ class BoundDeducer: public IRVisitor {
       // NOTE: this accounts for truc div behavior.
       bool target_is_non_neg = expr_map_[target_var].can_prove_non_negative();
 
-      if (is_greater_) {
+      if (comp_op == kGreater) {
         result_ += 1;
+      } else if (comp_op == kEqual) {
+        // condition unsatisfiable as with truc div, it will change the expression
```

Comment:
s/truc div/trunc div ?

Reply:
I can change it. but all other references use the word `truc` in this file and some other file.

Reply:
The first reference came in from 4273e461f and it seems like a typo for the use of (trunc div) short form for truncating divide. What semantics does 'truc div' actually have ?

Reply:
@yzhliu @u99127 Changed the `truc` to `trunc` in this file